projects/element-ng/chat-messages/si-ai-message.component.html

```diff
@@ -0,0 +1,43 @@
+<si-chat-message alignment="start" actionsPosition="bottom" [loading]="loading()">
+  @if (content()) {
+    @let content = textContent();
+    @if (content) {
+      <span class="text-pre-wrap">{{ content }}</span>
+    } @else {
+      <div #formattedContent> </div>
+    }
+  }
+
+  @if (actions().length > 0 || secondaryActions().length > 0) {
+    <div class="d-flex gap-4 ai-message-actions" siChatMessageAction>
+      @for (action of actions(); track $index) {
+        <button
+          type="button"
+          class="btn btn-ghost btn-circle btn-sm"
+          [attr.data-id]="action.id"
```

Comment:
I would remove this enrichment of IDs. We don’t do this as a general concept, and I don’t like to see it sneak in.